tests/unit_tests/experiment_plans/test_flyscan_xray_centre_plan.py

```diff
@@ -187,12 +194,14 @@ def test_results_adjusted_and_passed_to_move_xyz(
         run_gridscan: MagicMock,
         move_aperture: MagicMock,
         fake_fgs_composite: FlyScanXRayCentreComposite,
-        mock_subscriptions: XrayCentreCallbackCollection,
         test_fgs_params: GridscanInternalParameters,
-        RE: RunEngine,
+        RE_with_subs: tuple[RunEngine, XrayCentreCallbackCollection],
     ):
+        RE, mock_subscriptions = RE_with_subs
         set_up_logging_handlers(logging_level="INFO", dev_mode=True)
         RE.subscribe(VerbosePlanExecutionLoggingCallback())
+        for cb in list(mock_subscriptions):
+            RE.subscribe(cb)
```

Comment:
Should: Is this not adding the subscriptions twice given it's done in the fixture?

Reply:
yes